lib/bemxjst/match.js

```diff
@@ -30,7 +30,19 @@ MatchNested.prototype.exec = function exec(context) {
       return false;
   }
 
-  return val[this.keys[i]] === this.value;
+  var keyVal = val[this.keys[i]];
+  var thisVal = this.value;
+
+  if (
+      keyVal === undefined ||
+      keyVal === null ||
```

Comment:
I wish I could say that it is slowly but I can’t. [jsperf.com](jsperf.com) is down :-\